app/views/qa/show.html.erb

```diff
@@ -31,38 +30,60 @@
             } %>
           <% end %>
         <% elsif multiple_question? %>
-          <%= render "govuk_publishing_components/components/checkboxes", {
-            name: "#{current_facet['facet']['key']}[]",
-            heading: "Select all that apply",
-            visually_hide_heading: true,
-            items: options
-          } %>
+          <div class="govuk-!-margin-top-8">
+            <%= render "govuk_publishing_components/components/checkboxes", {
+              name: "#{current_facet['facet']['key']}[]",
+              heading: current_facet["question"],
+              description: render("govuk_publishing_components/components/govspeak", {
+                content: current_facet["description"].html_safe
```

Comment:
What happens if the current_facet doesn't have a description? I'm asking because `multiple_grouped_question` has a guard around `current_facet["description"]`.

Reply:
I've added a guard on this too now